core/src/main/java/jenkins/install/SetupWizard.java

```diff
@@ -126,6 +129,13 @@ public SetupWizard(Jenkins j) throws IOException, InterruptedException {
         } catch (ServletException e) {
             throw new AssertionError(e);
         }
+        
+        try {
+            // Make sure plugin metadata is up to date
+            UpdateCenter.updateDefaultSite();
+        } catch (Exception e) {
```

Comment:
Hm, I suppose so... mainly I don't want it to fail here, it will try again when the wizard is accessed if it hasn't had a successful site update. I might rather leave it as-is in case someone decides to change the `updateDefaultSite()` without considering it being used here.